config/default_jobs/weekday_jobs.yaml

```diff
@@ -0,0 +1,71 @@
+# Jobs that run daily at 1AM, typically getting the 2 most
+# recent years of data for critical tables
+addn:
+  table_name: iasworld.addn
+  # Update this year to update the rest of the minimum years in
+  # the file
+  min_year: &min_year 2023
```

Comment:
This diff moves the `min_year` anchor definition to the first occurrence in the file, rather than defining it in a dedicated `define`  block at the top of the file.